model/config.go

```diff
@@ -513,11 +513,6 @@ func (o *Config) SetDefaults() {
 		o.FileSettings.AmazonS3Endpoint = "s3.amazonaws.com"
 	}
 
-	if o.FileSettings.AmazonS3Region == "" {
```

Comment:
This was talked about in the master PR: https://github.com/mattermost/platform/pull/7010

> No if region is blank minio will use GetBucketLocation to get the appropriate region, if that fails then we look for x-amz-bucket-region, if that fails then we default to us-east-1